docs/pages/foundations/forms/available_components.js

```diff
@@ -0,0 +1,249 @@
+          showHeading={false}
+          defaultCode={` function Example(props) {
+            const [value, setValue] = React.useState('');
+
+            return (
+              <Box width={500}>
+                <TextField
+                  autoComplete="username"
+                  id="header-example"
+                  label="Username"
+                  onChange={({ value }) => {
+                    setValue(value);
+                  }}
+                  placeholder="Please enter your username"
+                  type="text"
+                  value={value}
+                />
+              </Box>
+            );
+          }
+        `}
+        />
+        <Flex direction="column" gap={4}>
+          <Text>
+            TextField is used for single lines of text. It also allows for other types of inputs
+            like passwords and Tags.
+          </Text>
+
+          <Text>
+            <Link href="/web/textfield" underline="always">
+              Go to the TextField component
+            </Link>
+          </Text>
+        </Flex>
+      </MainSection>
+      <MainSection name="TextArea">
+        <Example
+          id="textareaExample"
+          name="TextArea example"
+          showCode={false}
+          showHeading={false}
+          defaultCode={`function Example(props) {
+            const [value, setValue] = React.useState('')
+            return (
+              <Box width="100%">
+                <TextArea
+                  id="headerExample"
+                  onChange={({value}) => setValue(value)}
+                  placeholder="Write something about yourself..."
+                  label="About me"
+                  value={value}
+                />
+              </Box>
+            );
+          }`}
+        />
+        <Flex direction="column" gap={4}>
+          <Text>
+            TextArea is used for inputting longer, paragraph-length text. An example is asking users
+            to write out their opinion of a feature.
+          </Text>
+
+          <Text>
+            <Link href="/web/textarea" underline="always">
+              Go to the TextArea component
+            </Link>
+          </Text>
+        </Flex>
+      </MainSection>
+
+      <MainSection name="NumberField">
+        <SandpackExample
+          code={ref}
+          hideEditor
+          name="Main NumberField Example"
+          previewHeight={previewHeightPx}
+        />
+        <Flex direction="column" gap={4}>
+          <Text>NumberField allows for numerical input only.</Text>
+
+          <Text>
+            <Link href="/web/numberfield" underline="always">
+              Go to the NumberField component
+            </Link>
+          </Text>
+        </Flex>
+      </MainSection>
+      <MainSection name="Fieldset">
+        <Example
+          id="fieldsetExample"
+          name="FieldSet example"
+          showCode={false}
+          showHeading={false}
+          defaultCode={`function RadioButtonExample() {
```

Comment:
Can we use a Checkbox example? 

We should not recommend RadioButton but RadioGroup